index.rst

```diff
@@ -171,6 +171,10 @@
                <h2>News</h2>
 
                <ul>
+               <li>
+                  2021-03-15:
+                  We now have a Twitter handle <a href="https://twitter.com/gmt_dev">@gmt_dev</a>!
```

Comment:
```suggestion
                  GMT now has a Twitter handle <a href="https://twitter.com/gmt_dev">@gmt_dev</a>!
```
I prefer this phrasing, but feel free to comment if you think the more personal 'we' is better

Reply:
Let's go with yours :)